src/sbin/lorax-composer

```diff
@@ -240,10 +240,11 @@ if __name__ == '__main__':
 
     start_queue_monitor(server.config["COMPOSER_CFG"], uid, gid)
 
-    # Drop root privileges on the main process
```

Comment:
We should just cherry-pick over the rhel8 commit dbc4d08de40521ca8b1d64c3c0f36e6fc67be03c

Reply:
Ah that's much better. Done.